cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -181,6 +181,54 @@ class ReadFileTest {
         assertEquals(ion.singleValue(expected), actual)
     }
 
+    @Test
```

Comment:
These tests do test the different supported PartiQL CLI options, but they don't really show the difference between the formats (e.g. `readPostgreSQLTextFile` and `readMongodbTsvFile` have the same CSV string).

The documentation for CSVFormat's [POSTGRESQL_CSV](https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html#POSTGRESQL_CSV) states:

> This is a comma-delimited format with a LF character as the line separator. Values are double quoted and special characters are escaped with '"'. The default NULL string is "".

The documentation for CSVFormat's [MYSQL](https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html#MYSQL) states:

> This is a tab-delimited format with a LF character as the line separator. Values are not quoted and special characters are escaped with '\'. The default NULL string is "\\N".

I think using different CSV strings with those format nuances will provide better test cases for the different CSV formats.